src/test/java/com/faforever/client/game/TeamCardControllerTest.java

```diff
@@ -0,0 +1,75 @@
+package com.faforever.client.game;
+
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.player.Player;
+import com.faforever.client.replay.Replay.PlayerStats;
+import com.faforever.client.test.AbstractPlainJavaFxTest;
+import com.faforever.client.theme.UiService;
+import com.faforever.client.util.Rating;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableMap;
+import javafx.scene.control.Label;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TeamCardControllerTest extends AbstractPlainJavaFxTest {
+  private TeamCardController instance;
+  @Mock
+  private Player player1;
+  private ArrayList<Player> list;
```

Comment:
I'm sure we can find a better field name than `list`. And since it's not a mock, I'd move it down to the other non-mocks.